### PR TITLE
feat: svelte preprocessor for server-side rendering (SSR)

### PR DESCRIPTION
**Pull request by divdavem**

### CODE REVIEW

File: svelte/preprocess/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+							const value = extractValue(attribute);
+							str.overwrite(attribute.start, attribute.end, `use:${varPrefix}classDirective={${value}}`);
+							str.appendRight(end, `, [${varPrefix}classDirective, ${extractValue(attribute)}]`);
+							requiredImports.add('classDirective');
+						}
+
+						str.appendRight(end, `))}`);
+					}
+				}
+				item.children?.forEach(processItem);
```
Comment (andreascorti) [inline svelte/preprocess/src/index.ts:69]: wouldn't maybe better to use [walk](https://svelte.dev/docs/svelte-compiler#walk) for performances?

**Review by andreascorti — COMMENTED** [review]
thanks @divdavem for your work, waiting for the final version you are working on

File: svelte/preprocess/src/index.ts (hunk 1 above)
Comment (divdavem) [reply svelte/preprocess/src/index.ts:69]: `walk` does not seem to be compatible with the html part of svelte files, only the script parts.

**Review by quentinderoubaix — APPROVED** [review]
Thanks for the PR !
As discussed, will open another Issue for the things to add to the preprocessor (modifier, options + transforming our Svelte files in svelte/bootstrap)

**Comment by divdavem** [comment]
@andreascorti @quentinderoubaix Thank you for your review!

**Pull request merged**